scripts/FigS2B_adt-plots.R

```diff
@@ -0,0 +1,143 @@
+# This script is used to create the mini versions of the plots included in the ADT report
+# we use the results from SCPCS000216/SCPCL000290
+
+renv::load()
+
+library(ggplot2)
+library(SingleCellExperiment)
+
+theme_set(
+  theme_classic() +
+    theme(
+      strip.background = element_rect(fill = "transparent"),
+      # no axis ticks or labels
+      axis.line = element_blank(),
+      axis.ticks = element_blank(),
+      axis.text = element_blank(),
+      # add a square around each of the plots
+      panel.background = element_rect(colour = "black", linewidth=0.5),
+      aspect.ratio = 1
+    )
+)
+
+set.seed(2024)
+
+
+# Set up -----------------------------------------------------------------------
+
+# define file paths and read in sce objects
```

Comment:
I might define `sample_id` and `library_id` first and then use below? But for a one-off, doesn't seem super necessary. Up to you.